addons/hr_expense/static/src/views/list.js

```diff
@@ -28,27 +34,28 @@ export class ExpenseListController extends ExpenseDocumentUpload(ListController)
 
     displaySubmit() {
         const records = this.model.root.selection;
-        return records.length && records.every(record => record.data.state === 'draft') && this.isExpenseSheet;
+        return records.length && records.every(record => record.data.state === 'draft') && this.isExpenseSheet && !this.onMobile;
     }
 
     displayCreateReport() {
-        const records = this.model.root.selection;
```

Comment:
I don't know the JS modes so well, but I'm curious why this changes from `root.records` to `root.selection`, but not for the functions around this one

Reply:
There was some specs required related to this button, and that was one of them.

Reply:
![image](https://github.com/odoo/odoo/assets/160074741/cd022b20-9e89-4cab-9272-1bc84c1cd6eb)

Reply:
But what is the difference between `this.model.root.records` and `this.model.root.selection`?

It looks like `action_show_expenses_to_submit` is still using `this.model.root.selection` (which makes sense...)


____

Please don't resolve conversations if you didn't fix anything. Or if I didn't say that I agreed with you (or resolved it myself)

Reply:
records -> all records in the list view 
selection -> selected records of the list view

The method `action_show_expenses_to_submit` indeed uses selection by default. But the method it calls (i.e., `get_expenses_to_submit` of the model `hr_expense`) returns all draft expenses if none were selected in the list view

Reply:
What happens if you select only reported expenses?
Then it will submit a report for all the other unreported expenses?

Reply:
Creating 1 or 2 reports (if several payment_modes) with all unreported expenses of the user's employee if any, else raise